refinery/user_files_manager/views.py

```diff
@@ -1,21 +1,21 @@
 import logging
-from django.http import HttpResponseNotAllowed
+
+from rest_framework.views import APIView
 from rest_framework.response import Response
-from data_set_manager.utils import (
-    search_solr, format_solr_response,
-    generate_solr_params_for_user)
+
+from data_set_manager.utils import (search_solr, format_solr_response,
```

Comment:
Check out https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#python

The imports  in parens should be in alphabetical order